hack/test-end-to-end.sh

```diff
@@ -295,7 +295,7 @@ registry_pod=$(osc get pod | grep docker-registry | awk '{print $1}')
 osc exec -p ${registry_pod} whoami | grep root
 
 # Port forwarding
-osc port-forward -p ${registry_pod} 5001:5000 &
+osc port-forward -p ${registry_pod} 5001:5000  &> "${LOG_DIR}/port-forward.log" &
```

Comment:
No.  I can't link right to it, but that's "redirect stdout and stderr to file": http://linux.die.net/man/1/bash